src/pages/CreateEvent/EventForm.tsx

```diff
@@ -163,230 +232,373 @@ const EventForm: React.FC = () => {
+              <Controller
+                name="startDate"
+                control={control}
+                rules={{ required: "Start date is required" }}
+                render={({ field }) => (
+                  <input
+                    type="date"
+                    {...field}
+                    className={`form-control flex-1 ${
+                      errors.startDate ? "error" : ""
+                    }`}
+                  />
+                )}
+              />
+              <span className="divider">|</span>
+              <Controller
+                name="startTime"
+                control={control}
+                rules={{ required: "Start time is required" }}
+                render={({ field }) => (
+                  <input
+                    type="time"
+                    {...field}
+                    className={`form-control ${
+                      errors.startTime ? "error" : ""
+                    }`}
+                  />
+                )}
+              />
             </div>
+
+            {errors.startDate && (
+              <p className="error">{errors.startDate.message}</p>
+            )}
+            {errors.startTime && (
+              <p className="error">{errors.startTime.message}</p>
+            )}
           </div>
 
           <div className="sub-section">
-            <span>End *</span>
-            <div className="date-time-container">
-              <div className="date-input">
-                <CalendarIcon className="icon" />
-                <Controller
-                  name="endDate"
-                  control={control}
-                  rules={{ required: "End date is required" }}
-                  render={({ field }) => (
-                    <input
-                      type="date"
-                      {...field}
-                      className={`form-control ${
-                        errors.endDate ? "error" : ""
-                      }`}
-                    />
-                  )}
-                />
-              </div>
-
-              <div className="time-input">
-                <ClockIcon className="icon" />
-                <Controller
-                  name="endTime"
-                  control={control}
-                  rules={{ required: "End time is required" }}
-                  render={({ field }) => (
-                    <input
-                      type="time"
-                      {...field}
-                      className={`form-control ${
-                        errors.endTime ? "error" : ""
-                      }`}
-                    />
-                  )}
-                />
-              </div>
+            <span>End<sup>*</sup></span>
+            <div className="date-input-container">
+              <Controller
+                name="endDate"
+                control={control}
+                rules={{ required: "End date is required" }}
+                render={({ field }) => (
+                  <input
```

Comment:
should make unselectable dates that are before the start date and time

Reply:
Done